api/rest-client.ts

```diff
@@ -108,17 +122,16 @@ export class RingRestClient {
     try {
       const response = await requestWithRetry<AuthTokenResponse>({
         url: 'https://oauth.ring.com/oauth/token',
-        data: {
+        json: {
           client_id: 'ring_official_android',
           scope: 'client',
           ...grantData,
         },
-        method: 'POST',
+        method: 'post',
         headers: {
-          'content-type': 'application/json',
           '2fa-support': 'true',
           '2fa-code': twoFactorAuthCode || '',
-          hardware_id: await hardwareIdPromise,
+          'hardware_id': await hardwareIdPromise,
```

Comment:
This will probably cause a lint error.  You can check by running `npm run lint`

Reply:
I thought it is good practice to quote all JSON keys according to the standard? It isn't representing a property for the purpose of OOP after all. Although I can remove the quotes, that's not an issue.

Reply:
JSON does require that all keys be quoted, but javascript objects to not require it.  In plain javascript objects it works with or without quotes, unless you have special characters in the key (like the `-` in the keys above).  Most javascript linters will have quotes removed when possible since they unnecessary extra characters in this case.